openapi/schemas.yaml

```
@@ -258,7 +241,9 @@ components:
     limits-outgoing:
       title: limits-outgoing
       description: Open Payments specific property that defines the limits under which outgoing payments can be created.
-      type: object
+      type:
+        - object
+        - null
```

Based on examples, I'm seeing at least `null` (and maybe `object`) might need to be a "value"
```suggestion
        - 'null'
```
https://github.com/OAI/OpenAPI-Specification/issues/2244#issuecomment-899084964

Not sure if that will resolve the validator errors